### DISCUSSION TITLE
Cant access SO web interface - wrong analyst/host ip?

### DISCUSSION THREAD

**Discussion by Netrunner1883** [Security-Onion-Solutions/securityonion#10875] [Unsupported Versions]
Hi!
Quite new to this. Have went over the SO support section and gone one previous posts on this topic, without having figured it out. So hope anyone can help :) 

I used the official ISO image (2.3.260-20230620 ISO image) and followed the official installation guide on youtube.

Host hardware machine runs windows 10 pro, and the ISO image is run in VMware workstation 17.

When log on to the VM and run "so-status" all services respond with OK (green). 

When running "ipconfig" on the windows 10 pro host machine, I get the "ethernet adapter vmware network adapter vmnet1" as 192.168.195.1. This is the IP I have added as "so-allow" analyst role.

And this is where I might have done something wrong? When I try to access that address in the web browser from the windows 10 machine, the browser (firefox) responds with "unable to connect - the site could be unavailable or too busy". 

How do I know I have set the right manager/host address? 

I tried to run "sudo so-ip-update". It responded with "Found old IP 192.168.195.1. Detected new IP 192.168.195.1 on interface ens33. ERROR: IP address has not changed. Exiting.

**Comment by cm-ops** [comment]
What is your url_base? You can check it in `/opt/so/saltstack/local/pillar/global.sls` or run the following command:` salt-call pillar.get global:url_base`
